metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorConfig.java

```diff
@@ -0,0 +1,258 @@
+/*
+ *       Copyright 2017 Netflix, Inc.
+ *          Licensed under the Apache License, Version 2.0 (the "License");
+ *          you may not use this file except in compliance with the License.
+ *          You may obtain a copy of the License at
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *          Unless required by applicable law or agreed to in writing, software
+ *          distributed under the License is distributed on an "AS IS" BASIS,
+ *          WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *          See the License for the specific language governing permissions and
+ *          limitations under the License.
+ */
+
+package com.netflix.metacat.connector.hive;
```

Comment:
I still think we need to move all our connector packages under something like `com.netflix.metacat.connector.impl.*` so that we can safely ignore that package. Either that or ensure that we don't have anything in the `com.netflix.metacat.connector` package we want scanned by default. Which we should be able to do cause it's in common-server and we probably shouldn't auto wire anything that package anyway